### PR TITLE
Fix solo12 joint polarities. Update file to the one used in production at NYU

### PR DESCRIPTION
**Pull request by jviereck**
This fixes the joint polarities as they are used at NYU in production on the robot.

Fixes partially #1 .

### CODE REVIEW

File: demos/config_solo12.yaml [hunk 1]
```diff
@@ -4,36 +4,37 @@ robot:
         motor_numbers: [0, 3, 2, 1, 5, 4, 6, 9, 8, 7, 11, 10]
         motor_constants: 0.025
         gear_ratios: 9.
-        max_currents: 1.
+        max_currents: 8.
         reverse_polarities: [
-            true, false, true, true, false, false,
-            true, false, true, true, false, false
+            false, true, true, true, false, false,
+            false, true, true, true, false, false
```
Comment (paLeziart) [inline demos/config_solo12.yaml:10]: On our solo I have to use
```
true, true, true, false, false, false,
true, true, true, false, false, false
```
to be consistent with the urdf we are using (https://github.com/Gepetto/example-robot-data/tree/master/robots/solo_description/robots/solo12.urdf). To check that I loaded the robot using its urdf in a viewer (gepetto-viewer-corba) then I continuously set its configuration to robot.joints.positions. While sending 0 torques, I moved the legs by hand and verified that the urdf robot in the viewer moved in the same way (if the polarity is wrong the legs in the viewer move in the other direction than in reality).

I'm not sure what could explain our difference, maybe a urdf change or a different wiring on the robot ?

Comment (jviereck) [reply demos/config_solo12.yaml:10]: The motor numbers are the same for you? I will verify the urdf you linked in a bit. Also, I will do a live visualization and double check again.

Comment (jviereck) [reply demos/config_solo12.yaml:10]: Can you verify the wiring of solo? We use what we have specified on the wiki for the wiring order.

Comment (paLeziart) [reply demos/config_solo12.yaml:10]: > The motor numbers are the same for you? I will verify the urdf you linked in a bit. Also, I will do a live visualization and double check again.

Yes motor numbers are the same `[0, 3, 2, 1, 5, 4, 6, 9, 8, 7, 11, 10]`.



> Can you verify the wiring of solo? We use what we have specified on the wiki for the wiring order.

I'll ask Thomas about the wiring, I'm not sure how to check.

**Review by MaximilienNaveau — APPROVED** [review]
I guess I approve, just make sure the polarity story is fine!
I could check this branch on the Solo12 in MPI as well.

**Comment by jviereck** [comment]
> I could check this branch on the Solo12 in MPI as well.

Could you do this please? Please make sure you don't just test with a PD controller as you don't see if the orientation is right there. Best would be to move joints independently, log the data and verify the motion (that's what I did) or run one of our more advanced controllers like reactive stepping.

**Pull request merged**